src/pkgcheck/checks/python.py

```diff
@@ -237,6 +237,11 @@ def feed(self, pkg):
             return
 
         if eclass is None:
+            # virtual/pypy* need to be exempted as they serve as slot-matchers
+            # for other packages
+            if pkg.key in ('virtual/pypy', 'virtual/pypy3'):
```

Comment:
This tuple seems like it should be a class attr frozenset named something apt since more values could be added in the future.

Reply:
Added it to the global vars on top.